gaphas/tool/hover.py

```diff
@@ -1,11 +1,22 @@
-from gaphas.aspect import Finder
-from gaphas.tool.tool import Tool
+from gi.repository import Gtk
 
+from gaphas.tool.itemtool import handle_at_point, item_at_point
+from gaphas.types import Pos
+from gaphas.view import GtkView
 
-class HoverTool(Tool):
-    """Make the item under the mouse cursor the "hovered item"."""
 
-    def on_motion_notify(self, event):
-        view = self.view
-        pos = event.get_coords()[1:]
-        view.selection.set_hovered_item(Finder(view).get_item_at_point(pos))
+def hover_tool(view: GtkView):
+    ctrl = Gtk.EventControllerMotion.new(view)
+    ctrl.connect("motion", on_motion)
+    return ctrl
+
+
+def on_motion(ctrl, x, y):
+    # TODO: Check for claimed controller
```

Comment:
Do we need to do anything with this TODO before merging?

Reply:
I think this todo is already resolved.